test/QueryPlannerTest.cpp

```diff
@@ -25,7 +25,8 @@ TEST(QueryPlannerTest, createTripleGraph) {
         "SELECT ?x ?z \n "
         "WHERE \t {?x :myrel ?y. ?y ns:myrel ?z.?y xxx:rel2 "
         "<http://abc.de>}");
-    QueryPlanner qp(nullptr);
+    QueryPlanner qp(nullptr,
```

Comment:
At the top of this file, 
define (static or in anonymous namespace)
a single `cancellationHandle` and pass that to the queryPlanner.
Or even factor out the whole line that redundantly constructs the `QueryPlanner`.
Don't contribute to codebloat, even if it was prexisting.